inc/httputils.php

```diff
@@ -181,8 +189,8 @@ function http_rangeRequest($fh,$size,$mime){
     }
 
     // everything should be done here, exit (or return if testing)
-    if (defined('SIMPLE_TEST')) return;
```

Comment:
Why is `SIMPLE_TEST` check removed? If this is intended please check probably related tests.